src/Sidekick/App.xaml.cs

```diff
@@ -164,5 +172,33 @@ private void EnsureSingleInstance()
                 Current.Shutdown();
             }
         }
+
+        private void AttachErrorHandlers()
+        {
+            AppDomain.CurrentDomain.UnhandledException += (s, e) =>
```

Comment:
Just a note, this object has the `IsTerminating` flag, which tells when the exception is actually closing the app.